src/chapters/ldap-auth.xml

```diff
@@ -395,6 +395,15 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                             aliases.</para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-user-search-filter</property></term>
```

Comment:
OK - I built the manual from these changes and noticed that the organization of this section is slightly off. There's a section in the middle of this area which is devoted to users and search:

> ![ldap-user-search-stuff](https://cloud.githubusercontent.com/assets/4632905/24418078/9ab4ca40-139f-11e7-9e1b-e477588be48b.png)

but the new `ldap-user-search-filter` property documentation is simply appended to the end of the section among other unrelated properties:

> ![ldap-user-search-filter-section](https://cloud.githubusercontent.com/assets/4632905/24418107/ba15515c-139f-11e7-8405-2e61f2164435.png)

Perhaps this should be moved up a bit?